tests/project/test_projectPath.py

```diff
@@ -67,18 +68,18 @@ def test_path(self):
         )
 
     def test_root_path(self):
-        root_paths = self.settings_configuration["project_paths"]
+        root_paths = state.settings.configuration["project_paths"]
         self.assertIn(
-            self.project_path.root_path,
+            self.project_path.root_path[:-1],  # Trim the trailing "/"
             root_paths,
             msg="root project.root_path not properly set by default. Check if `project_check_enabled`.",
         )
 
     def test_project_path(self):
-        root_paths = self.settings_configuration["project_paths"]
+        root_paths = state.settings.configuration["project_paths"]
         self.assertIn(
             self.current_dir + "/test_project_path/",
```

Comment:
In your case, what is the ending of `self.current_dir` is it ending with a `/`? In that case it might be better to remove the tailing `/` rather than adding an additional one.